src/generator/AutoRest.Ruby/Model/PropertyRb.cs

```diff
@@ -1,15 +1,33 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using AutoRest.Core.Model;
+using AutoRest.Core;
+using AutoRest.Core.Utilities;
+using AutoRest.Extensions;
 
 namespace AutoRest.Ruby.Model
 {
     /// <summary>
     /// The model for property template.
     /// </summary>
-    public class PropertyRb : Property
+    public class PropertyRb : Core.Model.Property
     {
-        
+        protected PropertyRb()
+        {
+            // properties with dots by surrounding them with single quotes to represent 
```

Comment:
this comment seems to be missing something in the way it's written, let's update it